test/fw/ptl/utils/plugins/ptl_test_info.py

```diff
@@ -128,16 +129,22 @@ def wantClass(self, cls):
         """
         Is the class wanted?
         """
+        if not issubclass(cls, unittest.TestCase) or cls is PBSTestSuite:
```

Comment:
Hey @ericpershey We need to keep `or cls is unittest.Testcase` condition as well, along with condition for PBSTestSuite, as we don't want to display any info about unittest.Testcase.

Reply:
Updated.